modules/ipi-install-configuring-bios-for-worker-node.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc
+[id="configuring-bios-for-worker-node_{context}"]
+
+= Configuring BIOS for worker node
+
+During the installation process, the BIOS of the worker node can be configured
+by the following steps.
+
+.Procedure
+. Create manifests
```

Comment:
Add a period at the end. If there is an example, add a colon, and then provide an example of creating the manifests.